os/Makefile.unix

```diff
@@ -639,6 +639,7 @@ ifeq (download,$(firstword $(MAKECMDGOALS)))
 endif
 
 download:
+	$(Q) find ../ -name setting.txt -delete
```

Comment:
It is board specific file,  please move to `function pre_download()`

Reply:
I move it to predownload